swap_meet/vendor.py

```diff
@@ -0,0 +1,60 @@
+from .item import Item 
+
+#Wave 1
+# the attributes are the list of items/invetory and the class is vendor
+
+class Vendor:
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+        return
```

Comment:
so something weird happens when we pass in a list or dictionary as a default value. Every class instance made will *use* the exact same list or dictionary from the same memory space.

Instead we can try this:
```suggestion
    def __init__(self, inventory=None):
        if inventory == None:
            self.inventory = []  # now the list is assigned strictly to the instance made
        else:
            self.inventory = inventory
```